gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXRepo.cs

```diff
@@ -20,8 +20,8 @@ public static class GTXRepo
         /// <param name="repoPath">the repository main path</param>
         public static void Init(string repoPath)
         {
-            if (!Repository.IsValid(repoPath))
-                Repository.Init(repoPath, false);
+            if (!Repository.IsValid(repoPath))            
+             Global.error(repoPath  + " is not a valid git repo");            
```

Comment:
For what it's worth, almost all the AX programmers I've met have very little experience with version control at all, so requiring them to to the manual "git init" might be a usability problem - so I'm all for doing this inside gitax.

Also, the main use case is not going to be starting with an empty repo, but the first action is actually going to be cloning (I hope?), initing into an empty repo is going to be done only by the first developer on a particular project. So in the long term (not realy as a part of this patch) there should be actually several actions - "Clone", "init" and "connect to existing local repository".